yi-keymap-emacs/src/Yi/Keymap/Emacs/KillRing.hs

```diff
@@ -65,6 +65,11 @@ killLineB mbr = replicateM_ (fromMaybe 1 mbr) $ do
 killLine :: Maybe Int -> YiM ()
 killLine mbr = withCurrentBuffer (killLineB mbr) >> killringToClipboard
 
+-- | M-d
+-- | Like `killWordB`, but with system clipboard synchronization
+killWord :: YiM ()
+killWord = withCurrentBuffer (killWordB) >> killringToClipboard
```

Comment:
Maybe add the `repeatingArg` as `withCurrentBuffer (repeatingArg killWordB) >> killringToClipboard` ?

Reply:
Thx! You are right.